specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/ipTags.json

```diff
@@ -0,0 +1,420 @@
+            "schema": {
+              "$ref": "#/definitions/IpTagsListResult"
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ListByResourceGroup_IpTags": {
+            "$ref": "./examples/IpTagsListByResourceGroup.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/ipTags": {
+      "get": {
+        "tags": [
+          "IpTags"
+        ],
+        "operationId": "IpTags_List",
+        "description": "Gets all IpTags in a subscription.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of ipTag resources.",
+            "schema": {
+              "$ref": "#/definitions/IpTagsListResult"
+            }
+          },
+          "default": {
+            "description": "Unexpected error.",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List_IpTags": {
+            "$ref": "./examples/IpTagsListBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "IpTags": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/IpTagsPropertiesFormat",
+          "description": "Properties of the IpTags."
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
+        }
+      ],
+      "description": "The IpTags resource information."
+    },
+    "IpTagsListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IpTags"
+          },
+          "description": "The list of Iptags information resources."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of results."
+        }
+      },
+      "description": "Response for the ListIpTags API service call."
+    },
+    "IpTagsPropertiesFormat": {
+      "properties": {
+        "provisioningState": {
+          "$ref": "./network.json#/definitions/ProvisioningState",
```

Comment:
`provisioningState` should be read-only (it doesn't inherit this proeprty from network.json)

Reply:
Done